packages/internal/dex/sdk/src/constants/tokens/immutable-mainnet.ts

```diff
@@ -0,0 +1,33 @@
+import { IMMUTABLE_MAINNET_CHAIN_ID } from 'constants/chains';
+import { ERC20, Native } from '../../types';
+
+export const NATIVE_IMX_IMMUTABLE_MAINNET: Native = {
+  chainId: IMMUTABLE_MAINNET_CHAIN_ID,
+  decimals: 18,
+  symbol: 'IMX',
+  name: 'Immutable Token',
+  type: 'native',
+};
+
+export const WIMX_IMMUTABLE_MAINNET: ERC20 = {
+  chainId: IMMUTABLE_MAINNET_CHAIN_ID,
+  address: '0x3a0c2ba54d6cbd3121f01b96dfd20e99d1696c9d',
+  decimals: 18,
+  symbol: 'WIMX',
+  name: 'Wrapped Immutable Token',
+  type: 'erc20',
+};
+
+export const ETH_IMMUTABLE_MAINNET: ERC20 = {
```

Comment:
Should this be WETH?

Reply:
It's Eth from the contract/block explorer